<launch>
  <node name="davinci_needle_pose_publisher" pkg="cwru_davinci_grasp" type="davinci_needle_pose_publisher" respawn="false" output="screen"/>

  # specify which arm is going to perform needle grasp
  <arg name = "which_arm" default = "psm_one"/>

  # specify the name of the needle
  <arg name = "needle_name" default = "needle_r"/>

  # specify to pick or place needle
  <arg name = "movement" default = "pick"/>

  <node name="davinci_simple_needle_grasper_main" pkg="cwru_davinci_grasp" type="davinci_simple_needle_grasper_main" args="$(arg which_arm) $(arg needle_name) $(arg movement)" respawn="false" output="screen">
      <rosparam command="load" file="$(find cwru_davinci_grasp)/config/dvrk_psm_grasp_needle_data.yaml"/>
  </node>
</launch>
